packages/framework/client-logger/app-insights-logger/package.json

```diff
@@ -0,0 +1,83 @@
+{
+	"name": "@fluidframework/app-insights-logger",
+	"version": "2.0.0-internal.4.1.0",
+	"description": "Contains a Fluid logging client that sends telemetry events to Azure App Insights",
+	"homepage": "https://fluidframework.com",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/FluidFramework.git",
+		"directory": "packages/framework/client-logger/app-insights-logger"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"sideEffects": false,
+	"main": "dist/index.js",
+	"types": "dist/index.d.ts",
+	"scripts": {
+		"build": "npm run build:genver && concurrently npm:build:compile npm:lint && npm run build:docs",
+		"build:compile": "npm run build:tsc",
+		"build:docs": "api-extractor run --local && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../../_api-extractor-temp/",
+		"build:genver": "gen-version",
+		"build:test": "npm run build:compile",
+		"build:tsc": "npm run tsc",
+		"ci:build:docs": "api-extractor run && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../../_api-extractor-temp/",
+		"clean": "rimraf _api-extractor-temp coverage dist nyc *.tsbuildinfo *.build.log",
+		"eslint": "eslint src",
+		"eslint:fix": "eslint src --fix",
+		"format": "npm run prettier:fix",
+		"format-and-build": "npm run format && npm run build",
+		"format-and-compile": "npm run format && npm run build:compile",
+		"lint": "npm run prettier && npm run eslint",
+		"lint:fix": "npm run prettier:fix && npm run eslint:fix",
+		"prettier": "prettier --check . --ignore-path ../../../../.prettierignore",
+		"prettier:fix": "prettier --write . --ignore-path ../../../../.prettierignore",
+		"test": "npm run test:mocha",
+		"test:mocha": "mocha --recursive dist/test/**/*.spec.js -r node_modules/@fluidframework/mocha-test-setup --unhandled-rejections=strict",
+		"test:mocha:verbose": "cross-env FLUID_TEST_VERBOSE=1 npm run test:mocha",
+		"tsc": "tsc"
+	},
+	"dependencies": {
+		"@fluidframework/common-definitions": "^0.20.1",
+		"@microsoft/applicationinsights-web": "^2.8.11"
+	},
+	"devDependencies": {
+		"@fluidframework/mocha-test-setup": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
+		"@microsoft/api-extractor": "^7.34.4",
+		"@types/mocha": "^9.1.1",
+		"@types/sinon": "^7.0.13",
+		"concurrently": "^7.6.0",
+		"copyfiles": "^2.4.1",
+		"eslint": "~8.6.0",
+		"eslint-config-prettier": "~8.5.0",
+		"eslint-plugin-jest": "~27.1.5",
+		"eslint-plugin-react": "~7.28.0",
+		"eslint-plugin-react-hooks": "~4.6.0",
+		"mocha": "^10.2.0",
+		"prettier": "~2.6.2",
+		"rimraf": "^4.4.0",
+		"sinon": "^7.4.2",
+		"tinylicious": "0.7.2",
+		"ts-loader": "^9.3.0",
+		"tslib": "^1.10.0",
+		"typescript": "~4.5.5",
+		"webpack": "^5.76.2"
```

Comment:
`tinylicious` and `webpack` can probably go away, they don't seem used in this package anymore. A bit less familiar with `ts-loader` and `tslib`, but I suspect also unnecessary? `ts-loader` seems relevant to the webpack configuration that moved to the example package.

Reply:
`tslib` is required for app insights, but I removed `tinylicious` `webpack` & `ts-loader`